dora/core/common/src/main/java/alluxio/wire/WorkerIdentity.java

```diff
@@ -84,6 +118,28 @@ public String toString() {
         Parsers.getParserOfVersion(mVersion).getVersionSpecificRepresentation(this));
```

Comment:
so if this is used for serviceentityname which will eventaully becomes the path key on etcd, can u do "worker-%s" to avoid whitespaces

Reply:
the `serviceentityname` is only used for display to humans, right? will it be consumed programmatically and parsed into anything else?

Reply:
changed to no whitespace in [c41d385](https://github.com/Alluxio/alluxio/pull/18210/commits/c41d385d1c756e2e164e699ba66166db553076ef)